opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -406,12 +406,12 @@ def test_attributes(self):
             self.assertEqual(root.attributes["http.status_text"], "OK")
             self.assertEqual(root.attributes["misc.pi"], 3.14)
             self.assertEqual(root.attributes["attr-key"], "attr-value2")
-            self.assertEqual(root.attributes["empty-list"], [])
+            self.assertEqual(root.attributes["empty-list"], ())
             self.assertEqual(
-                root.attributes["list-of-bools"], [True, True, False]
+                root.attributes["list-of-bools"], (True, True, False)
```

Comment:
Probably you want to mutate the list after https://github.com/open-telemetry/opentelemetry-python/blob/0cf16478f47232d7c09c60d4415082e826cab59a/opentelemetry-sdk/tests/trace/test_trace.py#L395 to check that the code on this PR is working.